rpm/ovis-ldms-toss4.spec.in

```diff
@@ -257,6 +262,39 @@ package.
 %{_includedir}/*/*/*.h
 %{_includedir}/ovis-ldms-config.h
 
+%if %{with genders}
+%package genders
+Summary: LDMS base initscripts for libgenders control of %{name}
+Group: %{ldms_grp}
```

Comment:
I think we need a proper Group string here. I don't think "ldms_grp" is a thing.